### ISSUE TITLE
Tagbar integration overwrites user settings

### ISSUE THREAD

**Issue by ranebrown** [tbabej/taskwiki#194]
The tagbar settings applied [here](https://github.com/tbabej/taskwiki/blob/master/taskwiki/main.py#L344-L355) overwrite user tagbar settings in `.vimrc`.

It seems to me this should be removed and allow the user to handle the setup or at the very least add an option to disable this and make a note in the documentation.

**Comment by tbabej** [comment]
Agreed. This is a bit aggressive and should be controlled by a vim variable, defaulting to no integration.

**Comment by mountwebs** [comment]
This still overwrite user settings, if I'm not mistaken. I couldn't understand why tagbar was not working properly with vimwiki. I randomly tried disabling taskwiki, and it suddenly worked. That's when i came a cross this issue. I ended up editing the taskwiki-code to fix it.

Ps. I do love this plugin in all other regards!
